rpp30-boc-meltdown/biteshare
# Add facebook auth

## Description
This PR contains login to Biteshare app via facebook

## How to test
On Sign Up and Log In Screen, click the facebook icon. 
This will provide a pop up.
Enter your facebook email and password
Verify you are logged in to app .
![Screen Shot 2022-01-02 at 2 25 25 PM](https://user-images.githubusercontent.com/80971293/147887095-f01ea24d-faf8-4fe9-a68d-7eedc0484892.png)

## Notes


## Issue tracking
https://trello.com/c/dxbLcBt0
